docs/standard/library-guidance/index.md

```diff
@@ -23,18 +23,20 @@ Aspects of high-quality open-source .NET libraries:
 
 ## Recommendations
 
-With each article, there is a list of recommendations for your open-source .NET library using **Do**, **Consider**, **Avoid**, and **Do not**. The wording of each recommendation indicates how strongly it should be followed. For example, a **Do** recommendation is one that should almost always be followed:
+With each article, there is a list of recommendations for your open-source .NET library using **Do**, **Consider**, **Avoid**, and **Do not**. The wording of each recommendation indicates how strongly it should be followed.
```

Comment:
nit: While you're here, could you remove "open-source" from this sentence so that it is:

"With each article, there is a list of recommendations for your .NET library using "...

Many recommendations can apply to non-open-source libraries.